Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [package]
 name = "fuzzywuzzy"
 version = "0.0.2"
+edition = "2018"
```

Comment:
Will this cause any backwards compatibility issues if any library users are using an older edition of Rust?

Reply:
No, it is a crate local change. The ABI is the same, it just changes how rust parses *this* crate.